### ISSUE TITLE
ReactGridLayout vs. ResponsiveReactGridLayout

### ISSUE THREAD

**Issue by tculig** [STRML/react-grid-layout#872]
The documentation is not good when it comes to explaining the terminology. Here are my questions:
1. What's the difference between ReactGridLayout and ResponsiveReactGridLayout? What exactly is meant by "responsive"?
2. For ResponsiveReactGridLayout, what is meant by a "breakpoint"? Why do we need it and how should it be set?

**Comment by BlackSkyJY** [comment]
Same as above

**Comment by tculig** [comment]
So, to answer my own question, since no one else seems to give two craps:
1. The way I understand it ResponsiveGrid changes layout depending on the size of the screen. So you will have a different layout on mobile as opposed to desktop.
2. Breakpoints refer to resolutions at which another layout will take over. So if you set a breakpoint at say 768 pixels, you can set one layout bellow that breakpoint and another above. Anyone who has ever dabbed with Bootstrap and the like will know about this. 

In summary, these were questions that I actually knew the answers to, but it was not self-evident that this is what they are until I played a bit with the library.
